examples/dask/special-topics-rolling-averages.ipynb

```diff
@@ -0,0 +1,197 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "d21a962b-b3f8-49c6-bca9-6d07d3c5cb4c",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-01-25T17:33:22.602911Z",
+     "iopub.status.busy": "2022-01-25T17:33:22.602324Z",
+     "iopub.status.idle": "2022-01-25T17:33:25.666206Z",
+     "shell.execute_reply": "2022-01-25T17:33:25.665688Z",
+     "shell.execute_reply.started": "2022-01-25T17:33:22.602827Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "# Create Rolling Averages with Dask"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "866d3726-027e-4c04-ae8b-717683685472",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-01-25T17:33:22.602911Z",
+     "iopub.status.busy": "2022-01-25T17:33:22.602324Z",
+     "iopub.status.idle": "2022-01-25T17:33:25.666206Z",
+     "shell.execute_reply": "2022-01-25T17:33:25.665688Z",
+     "shell.execute_reply.started": "2022-01-25T17:33:22.602827Z"
+    },
+    "tags": []
+   },
+   "source": [
+    "\"I need to calculate a rolling average of a numerical column, in time series data. In pandas, I can do this with rolling(x).mean() with sorted values, but what do I do in Dask, with distributed data?\"\n",
+    "\n",
+    "* Sort by index within AND across partitions\n",
+    "* Know when to compute (convert to pandas DF) or persist (process computations on cluster)\n",
+    "* Run calculations, with attention to our need to cross partitions correctly.\n",
+    "\n",
+    "This example will walk you through these specific points, and demonstrate how it's done. We'll use New York City taxi trip data, and get the 30-day rolling average of base fare prices, for our example."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "622fb142-cee7-4659-a9b5-d9cbde6bf912",
+   "metadata": {},
+   "source": [
+    "## Single Node"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "c831ea89-1530-404f-8acc-94d3c28a603b",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "import dask\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()\n",
+    "\n",
+    "timeseries = dask.datasets.timeseries()\n",
+    "timeseries.rolling(\"1D\").mean().compute()"
+   ]
+  },
```

Comment:
I don't think there is a need to show a single node example (we don't ever do that on any other example)

```suggestion
```